packages/sui-bundler/shared/define.js

```diff
@@ -1,14 +1,9 @@
 const webpack = require('webpack')
 
-let pwd = process.cwd()
-
 // hack for Windows, as process.env.PWD is undefined in that environment
 // https://github.com/mrblueblue/gettext-loader/issues/18
-// Moreover, to have same caseing we need to map it.
 if (process.platform === 'win32') {
-  const [driveLetter, path] = pwd.split(':')
-  pwd = [driveLetter.toLowerCase(), path].join(':')
-  process.env.PWD = pwd
+  process.env.PWD = process.cwd()
```

Comment:
Without linking process.cwd() still returns "C:", but the rest of the paths are now in "C:" too. I supposed it has to do sth with linking...